examples/all-clusters-app/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -101,7 +101,7 @@ void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath &
 {
     TaskHandle_t task = xTaskGetCurrentTaskHandle();
     const char * name = pcTaskGetName(task);
-    if (!strcmp(name, "CHIP"))
+    if (strcmp(name, "CHIP"))
     {
         ESP_LOGE("all-clusters-app", "Attribute changed on non-Matter task '%s'\n", name);
```

Comment:
@bzbarsky-apple Can you check if including `esp_log.h` is required? Someone mentioned an issue: 

```
../main/CHIPDeviceManager.cpp: In function 'void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath&, uint8_t, uint8_t, uint16_t, uint8_t*)':



../main/CHIPDeviceManager.cpp:106:9: error: 'ESP_LOGE' was not declared in this scope



         ESP_LOGE("all-clusters-app", "Attribute changed on non-Matter task '%s'\n", name);



         ^~~~~~~~



../main/CHIPDeviceManager.cpp:106:9: note: maximum limit of 1000 namespaces searched for 'ESP_LOGE'



../main/CHIPDeviceManager.cpp:106:9: note: suggested alternative: 'ESP_LWIP'



         ESP_LOGE("all-clusters-app", "Attribute changed on non-Matter task '%s'\n", name);



         ^~~~~~~~

```

Reply:
It's possible!  This is compiling for me, but something could be pulling in the header when it shouldn't....